src/result.rs

```diff
@@ -118,7 +118,7 @@ pub enum OperationError {
     #[error("invalid glob pattern found in ignore list: {0}")]
     InvalidGlobPattern(ignore::Error),
     /// Parse error occurred while parsing the input.
-    #[error("failed to parse {input:?}")]
+    #[error("failed to parse {input}")]
```

Comment:
This fixes the `failed to parse Real("...")` error detail that was reported in the linked issue to instead display the correct error (e.g. `failed to parse /home/caleb/dev/rustfmt-sandbox/bad-syntax/src/openbrace.rs` and `failed to parse <stdin>` for stdin)